File: 2.Print_Hello_10_times/hello10.go
package main

import (
	"fmt"
	"strings"
)

func main() {
	fmt.Printf(strings.Repeat("Hello, Spaceack!\n", 5))
	for i := 0; i < 5; i++ {
		fmt.Printf("Hello, Spaceack!\n")
	}
}

File: 2.Print_Hello_10_times/hello10.js
function hello10(){
    console.log("Hello, Spaceack!\n".repeat(2))
    for (let i = 0; i < 3; i++) {
        console.log("Hello, Spaceack!");
      }
    [...Array(2)].forEach(() => console.log('Hello, Spaceack!'))
    let count = 0;
    while (count < 3) {
    count++; 
    console.log('Hello, Spaceack!');
    };
}

hello10()